payload-examples/api.github.com/index.json

```diff
@@ -56347,7 +56347,7 @@
             "type": "User",
             "site_admin": false
           },
-          "body": "Maybe you should use more emojji on this line.",
+          "body": "Maybe you should use more emoji on this line.",
```

Comment:
technically this really misspelled in github, but as this is a mock, i don't think it has to match the real world.